kong/plugins/statsd/log.lua

```diff
@@ -274,6 +278,54 @@ if ngx.config.ngx_lua_version >= 10011 then
       end
     end
   end
+  metrics.lmdb_usage = function (_, message, metric_config, logger, conf, tags)
+    -- we don't need this for every request, send every 1 minute
+    -- also only one worker needs to send this because it's shared
+    if worker_id ~= 0 then
+      return
+    end
+    if kong.configuration.database ~= "off" then
+      return
+    end
+
+    local now = ngx_time()
+    if lmdb_metrics_last_sent + LMDB_METRICS_SEND_THRESHOLD < now then
+      lmdb_metrics_last_sent = now
+      local lmdb_info, err = get_lmdb_info()
+      if err then
+        kong.log.err("failed to get lmdb info: ", err)
+        return
+      end
+      if conf.tag_style then
+        local tags = {
+          ["node"] = hostname,
+        }
+        logger:send_statsd("lmdb.used_space",
+          lmdb_info.last_used_page * lmdb_info.page_size, logger.stat_types.gauge,
+          metric_config.sample_rate, tags, conf.tag_style)
+        logger:send_statsd("lmdb.capacity",
+          lmdb_info.map_size, logger.stat_types.gauge,
+          metric_config.sample_rate, tags, conf.tag_style)
+      else
+        if conf.hostname_in_prefix then
+          logger:send_statsd("lmdb.used_space",
```

Comment:
Calculate metrics name `local metric_name = conf.hostname_in_prefix and "lmdb.used_space" or string_format("node.%s.lmdb.used_space", hostname)` or use an `if` statement to assign it, then remove the duplication of the `logger:send_statsd` lines.